sympy/core/relational.py

```diff
@@ -124,6 +124,31 @@ def reversed(self):
         a, b = self.args
         return Relational.__new__(ops.get(self.func, self.func), b, a)
 
+    @property
+    def reversedsign(self):
+        """Return the relationship with signs reversed.
+
+        Examples
+        ========
+
+        >>> from sympy import Eq
+        >>> from sympy.abc import x
+        >>> Eq(x, 1)
+        Eq(x, 1)
+        >>> _.reversedsign
+        Eq(-x, -1)
+        >>> x < 1
+        x < 1
+        >>> _.reversedsign
+        -x > -1
+        """
+        a, b = self.args
+        if not (isinstance(a, BooleanAtom) or isinstance(b, BooleanAtom)):
+            ops = {Gt: Lt, Ge: Le, Lt: Gt, Le: Ge}
+            return Relational.__new__(ops.get(self.func, self.func), -a, -b)
```

Comment:
Are there other possibilities for self.func than listed in ops? I would expect to use ops[self.func] if there aren't. If there are other possibilities does it make sense to fallback on using self.func?

Reply:
Eq and Ne are the two other possibilities.

Reply:
Oh okay. I'd find the code clearer if those were listed in ops.

Reply:
I added those and did the same in `reversed`.